docs/jplspec/jplspec.rst

```diff
@@ -213,108 +213,101 @@ to query these directly.
 
    >>> from astroquery.jplspec import JPLSpec
    >>> import astropy.units as u
-   >>> response = JPLSpec.query_lines_async(min_frequency=100 * u.GHz,
-   ...                                      max_frequency=1000 * u.GHz,
-   ...                                      min_strength=-500,
-   ...                                      molecule="H2O",
-   ...                                      parse_name_locally=True)
+   >>> result = JPLSpec.query_lines(min_frequency=100 * u.GHz,
+   ...                              max_frequency=1000 * u.GHz,
+   ...                              min_strength=-500,
+   ...                              molecule="H2O",
+   ...                              parse_name_locally=True)
+   >>> print([1, 2, 3])  # doctest: +ELLIPSIS
```

Comment:
is this for debug? Also, ELLIPSIS should be opted-in by default. If not that's an infrabug that we should fix on the config level.

Reply:
(Oh, now I see that my comment in the issue was ambiguous, I only meant to use `...` in the outputs for ELLIPSIS, but if you do that it should work out of the box.

Reply:
Oops!  I was having trouble with this test, and I thought it was about getting ... to work, so I thew in a debug test to figure it out.  It turns out I was printing the result table, but the test string was from `__repr__`.  Then I forgot to remove the debug test.  I'll fix that and verify that the directive isn't needed here.